codecs/numcodecs.fixedscaleoffset/README.md

```diff
@@ -0,0 +1,46 @@
+# FixedScaleOffset codec
+
+Defines a `array -> array` codec that performs a numeric transformation.
+
+## Codec name
+
+The value of the `name` member in the codec object MUST be `numcodecs.fixedscaleoffset`.
+
+## Configuration parameters
+
+See https://numcodecs.readthedocs.io/en/stable/filter/fixedscaleoffset.html for details about the configuration parameters.
+
+- `offset`
+- `scale`
+- `dtype`
+- `astype` (optional)
```

Comment:
the permitted values for the data types needs to be made clear, e.g. that they must be the JSON form of zarr v3 data types